starter-code/public/scripts/controllers/articleController.js

```diff
@@ -3,10 +3,14 @@ var app = app || {};
 
 (function(module) {
   const articleController = {};
-  // TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
+  // DONE: Setup a function that kicks off the fetching and rendering of articles, using the same
   // code that used to be in index.html.
   // Also be sure to hide all the main section elements, and reveal the #articles section:
-
-
+  articleController.setContent = function() {
+    app.Article.fetchAll(function() {
```

Comment:
Great job making sure fetchAll has a callback, but the hide and show should not be the callback.They should just be called after the app call above(not in a function)! For the first part of the todo, the fetchAll fetches the articles. But what do we call to render the articles? Hint Hint: Look at previous labs when we called fetchAll inside of the HTML.